sdk/aqueduct/utils/local_data.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import Optional
+
+from aqueduct.constants.enums import ArtifactType, LocalDataTableFormat
+from aqueduct.models.local_data import LocalData
+
+from ..error import InvalidUserArgumentException
+
+# The current local data maximum is temporarily set to 1MB.
+MAXIMUM_LOCAL_DATA_SIZE = 1000000
+
+
+def Local_Data(
+    path: str,
+    artifact_type: ArtifactType,
+    format: Optional[str] = None,
+) -> LocalData:
+    """Identify the local data which can be passed in as a parameter.
+
+    Args:
+        path:
+            The path to the data.
+        artifact_type:
+                The expected type of the data. The `ArtifactType` class in `enums.py` contains all
+                supported types, except for ArtifactType.UNTYPED.
+        format:
+                If the artifact type is ArtifactType.TABLE, the user has to specify the table format.
+                We currently support JSON, CSV, and Parquet.
+
+    Returns
+        A `LocalData` object which contains necessary information to create a parameter artifact.
```

Comment:
nit: "A configuration object which you can use to reference data in the local filesystem."